### PR TITLE
chore: surface image SHA with happy commands in GH job summary for easy copy + paste

### PR DESCRIPTION
**Pull request by seve**
I'm sick of grepping the job logs for the sha to use in happy commands, this should make it easier

### CODE REVIEW

File: .github/workflows/push-rdev.yml [hunk 1]
```diff
@@ -43,3 +43,8 @@ jobs:
         shell: bash
         run: |
           happy push --aws-profile "" --tag sha-${GITHUB_SHA:0:8}
+      - name: Create Summary With Happy Commands
+        run: |
+          echo "### Happy Commands :rocket:" >> $GITHUB_STEP_SUMMARY
```
Comment (seve) [inline .github/workflows/push-rdev.yml:48]:
```suggestion
          echo "### Happy Commands :rocket: :upside_down_face:" >> $GITHUB_STEP_SUMMARY
```

File: .github/workflows/push-rdev.yml [hunk 2]
```diff
@@ -43,3 +43,8 @@ jobs:
         shell: bash
         run: |
           happy push --aws-profile "" --tag sha-${GITHUB_SHA:0:8}
+      - name: Create Summary With Happy Commands
+        run: |
+          echo "### Happy Commands :rocket: :upside_down_face:" >> $GITHUB_STEP_SUMMARY
+          echo "* `happy create <stack-name> --tag ${GITHUB_SHA:0:8} --create-tag=false`" >> $GITHUB_STEP_SUMMARY
+          echo "* `happy update <stack-name> --tag ${GITHUB_SHA:0:8} --create-tag=false`" >> $GITHUB_STEP_SUMMARY
```
Comment (danieljhegeman) [inline .github/workflows/push-rdev.yml:50]:
```suggestion
          echo "* `happy create <stack-name> --tag ${GITHUB_SHA:0:8} --create-tag=false --skip-check-tag`" >> $GITHUB_STEP_SUMMARY
          echo "* `happy update <stack-name> --tag ${GITHUB_SHA:0:8} --create-tag=false --skip-check-tag`" >> $GITHUB_STEP_SUMMARY
```

Comment (seve) [reply .github/workflows/push-rdev.yml:50]: image

Comment (seve) [reply .github/workflows/push-rdev.yml]: image

**Review by danieljhegeman — APPROVED** [review]

**Pull request merged**